components/observations/SimpleForm.tsx

```diff
@@ -62,6 +63,7 @@ export const SimpleForm: React.FC<{
   const scrollViewRef = useRef<ScrollView>(null);
 
   const {nationalAvalancheCenterHost} = React.useContext<ClientProps>(ClientContext);
+  const today = new Date();
```

Comment:
When using the date picker, shouldn't this be something we pass in?

Reply:
For a new obs, I didn't think so, unless we want to let people pick dates that aren't today? Seems like a Charlotte question I guess 🤷.